src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.h

```diff
@@ -7,12 +7,18 @@
 struct ObjWarp2block;
 
 typedef void (*ObjWarp2blockActionFunc)(struct ObjWarp2block*, GlobalContext*);
+typedef s32 (*OtherFunc)(struct ObjWarp2block*, GlobalContext*);
 
 typedef struct ObjWarp2block {
     /* 0x0000 */ DynaPolyActor dyna;
     /* 0x0164 */ ObjWarp2blockActionFunc actionFunc_164;
-    /* 0x0168 */ ObjWarp2blockActionFunc actionFunc_168;
-    /* 0x016C */ char unk_16C[0xC];
+    /* 0x0168 */ OtherFunc actionFunc_168;
```

Comment:
Imo this function isn't anything like an action function, so it should be left as `func_168` or something. I also suggest renaming `actionFunc_164` to just `actionFunc`